core/src/main/kotlin/org/evomaster/core/problem/util/ParamUtil.kt

```diff
@@ -30,6 +38,69 @@ class ParamUtil {
 
         private val log: Logger = LoggerFactory.getLogger(ParamUtil::class.java)
 
+
+        /**
+         * bind values between [call] and [dbActions]
+         * [candidates] presents how to map [call] and [dbActions] at Gene-level
+         * [forceBindParamBasedOnDB] is an option to bind params of [call] based on [dbActions]
+         */
+        fun bindCallWithDBAction(
+            call: RestResourceCalls,
+            dbActions: MutableList<DbAction>,
+            candidates: MutableMap<RestAction, MutableList<ParamGeneBindMap>>,
```

Comment:
this needs to be refactored. RestAction is a mutable object, isn't it? if so, should not be used as a key in a map, especially as input to a public function

Reply:
actually, the `keys` of `candiates` are rest actions in `call`. here I want to bind the values based on the map provided by `candidates`.